stories/Chip.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { text, withKnobs } from '@storybook/addon-knobs';
+import { Chip, IconMatch } from '@textkernel/oneui';
+
+storiesOf('Chip', module)
+    .addDecorator(withKnobs)
+    .add('Chip', () => (
+        <Chip title={text('Title', 'This is a title that uses native browser functionality')}>
+            <IconMatch />
+            {text('Content', ' This is a chip')}
```

Comment:
Without the space the icon padding is missing. Would it make sense to support icons natively for a component like this? Or at least change the example to `<IconMatch />&nbsp;`.

Reply:
I think there is a point in that. I would however suggest to make a component that renders an icon and adds padding on right or left. Then that can be more easily reused in any component or just text. If we add it in the context of every component, it is more complicated, I believe, with at least 2 or 3 props needed... If you agree, I will make a ticket for it in ONEUI.